core/src/components/textarea/test/basic/index.html

```diff
@@ -67,12 +67,6 @@
             <ion-label color="primary">Clear on Edit</ion-label>
             <ion-textarea clear-on-edit="true"></ion-textarea>
           </ion-item>
-
-          <!-- TODO: Re-add auto grow with PR#24205 -->
-          <!-- <ion-item>
-            <ion-label color="primary">Autogrow</ion-label>
-            <ion-textarea auto-grow="true"></ion-textarea>
-          </ion-item> -->
```

Comment:
Why was this disabled?

Reply:
When migrating the Puppeteer tests to Playwright, I ran into issues with consistently capturing the screenshots because of auto-grow. We discussed disabling those auto grow screenshots until this PR was to be merged. 

When Amanda reviewed this PR, she called out that this test is effectively a duplicate of behavior captured in the other auto grow tests, so we elected for removing it entirely from the basic tests.